hazelcast/include/hazelcast/client/protocol/ClientMessage.h

```diff
@@ -619,6 +619,36 @@ class HAZELCAST_API ClientMessage
         return result;
     }
 
+    template<typename T>
```

Comment:
I'm pretty sure the function immediately above this one used to be used, when `std::is_trivial<boost::uuids::uuid>::value` was true

Reply:
Actually I just realized it might be less patch to just edit `std::is_trivial<typename T::value_type::first_type>::value` to `(std::is_trivial<typename T::value_type::first_type>::value || std::is_same<typename T::value_type::first_type, boost::uuid::uuid>::value)` above, if you would prefer something like that let me know,